packages/mdc-drawer/temporary/mdc-temporary-drawer.scss

```diff
@@ -65,7 +65,7 @@
 
     @include mdc-slideable-drawer;
 
-    @include mdc-rtl(".mdc-temporary-drawer__drawer") {
+    @include mdc-rtl {
```

Comment:
It looks like this actually just needs the correct root selector instead of the `__drawer` element.

This would be more specific in its output:
```css
@include mdc-rtl(".mdc-temporary-drawer") {
  ...
}
```

Once that's in I think were good to merge this.

Thanks!